person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  def initialize(_id, name, _age)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = 10
+  end
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  attr_writer :name
```

Comment:
- [ ]  `:name` and `:age` instance variables should be able to be modified, kindly consider using attr_accessor here for `:name`, and `:age`